rtf_vis_tool/src/stylesheets/LandingPageGraph.css

```diff
@@ -0,0 +1,159 @@
+/* Stylings for LandingPageGraph.js Component */
+
+/* This container takes up 100% of the screen space.
+ Sets relative positioning so that all nodes inside this primary container can be positioned absolutely 
+   (meaning with percent offsets from the top and left side of the screen).
+*/
+.lp_graph_container {
+    display: flex;
+    flex-direction: column;
+    position: relative;
+    width: 100%;
+    height: 100vh;
+}
+
+/*  Container for  the 'GTSFM Computational Graph Visualizer' header with tan background, black text, and large font.
+    The header has a black border with a width that spans the entire screen.
+*/
+.navbar {
+    height: 5%;
+    width: 100%;
+    background-color: #D2B48C;
```

Comment:
Thanks for all the updates, Adi.

One note here -- "tan" is not my favorite background color -- could you use one of the softer Tango color tones instead? https://en.wikipedia.org/wiki/Tango_Desktop_Project (as background)